Marsiple/Common/UI/TitleAndBodyView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  PostView.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 20/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+class TitleAndBodyView: UIView {
+    private let titleLabel = UILabel.autolayoutView()
+    private let bodyLabel = UILabel.autolayoutView()
+    var titleLabelFontSize: CGFloat = 20 {
+        didSet {
+            titleLabel.font = .systemFont(ofSize: titleLabelFontSize)
+        }
+    }
+    var bodyLabelFontSize: CGFloat = 15 {
+        didSet {
+            bodyLabel.font = .systemFont(ofSize: bodyLabelFontSize)
+        }
```

Comment:
The `didSet` code's quite short, what do you think about using a one-liner here:
`didSet { bodyLabel.font = .systemFont(ofSize: bodyLabelFontSize) }`